### PR TITLE
Added Largest Sum Contiguous Subarray [Python]

### LINKED ISSUE #313: Largest Sum Contiguous Subarray [Python]
#### This issue is about [Largest Sum Contiguous Subarray](http://www.geeksforgeeks.org/largest-sum-contiguous-subarray/).

- [x] Issue Title is in form `Algo/DS Name [Language]`.
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This Algo/DS is worth adding to this repository.
- [x] I ensure that I'm not assigned to 2 or more issues.
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

### PR DESCRIPTION
**Pull request by Monal5031**
Fixes #313

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

File: largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    :param arr: List of numbers to form subarray from.
+    :return: The maximum sum in all subarrays.
+    """
+    max_now = 0
+    max_next = 0
+    for i in arr:
+        max_next += i
+        if max_next > max_now:
+            max_now = max_next
```
Comment (aashutoshrathi) [inline largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py]: Change it to `max_now = max(max_now, max_next)`

File: largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        if max_next < 0:
```
Comment (aviaryan) [inline largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py]: Same can be done here.

File: largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py (hunk 1 above)
Comment (Monal5031) [inline largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py]: Done.

File: largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py (hunk 2 above)
Comment (Monal5031) [inline largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py]: Done.

File: largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        max_next += i
+        max_now = max(max_next, max_now)
+        max_next = max(0, max_next)
+    return max_now
+
+
+def main():
+    arr = [-2, -3, 4, -1, -2, 1, 5, -3]
+    print('Maximum contiguous sum is ' +
+          str(largest_sum_contiguous_subarray(arr)))
```
Comment (aashutoshrathi) [inline largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py]: Do we need `Maximum contiguous sum is`. Can't we just `print(largest_sum_contiguous_subarray(arr))` ?

Comment (Monal5031) [inline largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py]: Displaying a number wasn't looking good so I added some string before it,

Comment (aviaryan) [inline largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py]: Use `print('Maximum contiguous sum is', largest_sum_contiguous_subarray(arr))`. Much cleaner.

File: largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    for i in arr:
+        max_next += i
+        max_now = max(max_next, max_now)
+        max_next = max(0, max_next)
+    return max_now
+
+
+def main():
+    arr = [-2, -3, 4, -1, -2, 1, 5, -3]
+    print('Maximum contiguous sum is ', largest_sum_contiguous_subarray(arr))
```
Comment (aviaryan) [inline largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.py]: Remove space at the end of first parameter. (`sum is '`) should be (`sum is`)

**Pull request merged**